src/wix/WixToolset.Core/CommandLine/VersionCommand.cs

```diff
@@ -14,15 +14,19 @@ internal class VersionCommand : ICommandLineCommand
 
         public bool StopParsing => true;
 
-        public Task<int> ExecuteAsync(CancellationToken cancellationToken)
+        private IServiceProvider ServiceProvider { get; }
+
+        private VersionCommand() { }
+
+        public VersionCommand(IServiceProvider serviceProvider)
         {
-            // $(GitBaseVersionMajor).$(GitBaseVersionMinor).$(GitBaseVersionPatch)$(GitSemVerDashLabel)+$(Commit)
-            Console.WriteLine("{0}.{1}.{2}{3}+{4}", ThisAssembly.Git.BaseVersion.Major
-                                                  , ThisAssembly.Git.BaseVersion.Minor
-                                                  , ThisAssembly.Git.BaseVersion.Patch
-                                                  , ThisAssembly.Git.SemVer.DashLabel
-                                                  , ThisAssembly.Git.Commit);
+            this.ServiceProvider = serviceProvider;
+        }
 
+        public Task<int> ExecuteAsync(CancellationToken cancellationToken)
+        {
+            var branding = this.ServiceProvider.GetService<IWixBranding>();
+            Console.WriteLine(branding.ReplacePlaceholders("[ProductVersion]"));
```

Comment:
I think [my reply above](https://github.com/wixtoolset/wix4/pull/74/files#r778878727) got lost in all the other communication.

For this fix, there shouldn't be any changes to the `VersionCommand.cs` .